presto-thrift-testing-server/src/main/java/com/facebook/presto/connector/thrift/server/TpchServerUtils.java

```diff
@@ -0,0 +1,53 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.connector.thrift.server;
+
+import com.facebook.presto.spi.type.Type;
+import com.facebook.presto.tpch.TpchMetadata;
+import io.airlift.tpch.TpchColumnType;
+import io.airlift.tpch.TpchTable;
+
+import java.util.List;
+
+import static com.facebook.presto.tpch.TpchMetadata.getPrestoType;
+import static java.util.stream.Collectors.toList;
+
+final class TpchServerUtils
```

Comment:
This should not be a separate class and instead just be inside of `ThriftTpchService`